ui/src/conferences/containers/SearchPageContainer.jsx

```diff
@@ -50,7 +55,7 @@ function ConferenceSearchPage({
     <>
       <DocumentHead title={TITLE} description={META_DESCRIPTION} />
       <Row>
-        <Col xs={24} lg={22} xl={20} xxl={18}>
+        <Col {...SEARCH_PAGE_COL_SIZE_WITH_FACETS}>
```

Comment:
What is the reason of not utilizing your utility function here and you directly assign that one ? We do not need the SEARCH_PAGE_COL_SIZE_NO_RESULTS here in case there are no results ?

Reply:
it is due to difference in layout in those two pages - facets are always visible, no matter if there are results or not, unlike to other pages. my function takes numberOfResults as first argument to decide which size to display, I could use it here as ```...columnSize(1, true)``` (use fake number of results) to make it work, but I think in those two cases it's better to use ```SEARCH_PAGE_COL_SIZE_WITH_FACETS``` directly

Reply:
ok, thanks for clarifying, we can have it like this